.gitignore

```diff
@@ -22,3 +22,4 @@ debug
 /win_install.log
 /linux_install.log
 /src/default_excludes.mk
+.vscode
```

Comment:
You are not allowed to change or check in the .gitignore file. Also all files must terminate with a \n